protractor.conf.js

```diff
@@ -64,8 +69,15 @@ exports.config = {
   suites: {
     e2e: globby.sync([
       './src/test-e2e/**/*-e2e.spec.ts',
-      '!./src/test-e2e/login/*-sso-e2e.spec.ts',
-      '!' + checkSuiteGlob
+      ...excludeTests
+    ]),
+    e2ePart1: globby.sync([
+      './src/test-e2e/[a-d]*/**/*-e2e.spec.ts',
```

Comment:
I wonder if rather than doing this alphabetically, we explicitly define the 2nd suite - we'll call that the long suite - with (e.g.):

- Service tests
- App deploy tests - GitHub and Local File

Then we define the 1st suite as being all test file with an exclusion on all of the tests in the 2nd suite.